src/services/actionCableSocketWrangler.js

```diff
@@ -16,6 +16,12 @@ function($rootScope, ActionCableWebsocket, ActionCableConfig, ActionCableControl
   var _live= false;
   var _connecting= false;
   var _reconnectTimeout= false;
+  var preConnectionCallbacks= [];
```

Comment:
This is unused

Reply:
Sorry about that, got pulled in from the other branch somehow :sweat_smile: